src/rime/config/config_compiler.cc

```diff
@@ -181,7 +186,8 @@ static bool EditNode(an<ConfigItemRef> target,
   string path = StripOperator(key, appending || merging);
   DLOG(INFO) << "appending: " << appending << ", merging: " << merging
              << ", path: " << path;
-  return TraverseCopyOnWrite(target, path, writer);
+  auto cow_node = merge_tree ? &TypeCheckedCopyOnWrite : &TraverseCopyOnWrite;
```

Comment:
The variable name sounds like a "node" but it's not. Name it using a verb phase, for example "find_target_node"?